css/src/main/scala/io/udash/css/CssText.scala

```diff
@@ -12,6 +12,6 @@ trait CssText {
 object CssText extends CssText {
   private final class TextStyleModifier(styles: CssStyle*) extends Modifier {
     override def applyTo(t: Builder): Unit =
-      styles.foreach { s => t.appendAttr(cls.name, Builder.GenericAttrValueSource(s.classNames.mkString(" ", " ", ""))) }
+      styles.foreach { s => t.appendAttr(cls.name, Builder.GenericAttrValueSource(s.classNames.mkString)) }
```

Comment:
`classNames` should be separated by a space - please consider the following test case:
```scala
val fontAwesomeStyle = CssStyleNameWithSharedCompanion("far", "fa", "fa-grin")
i(fontAwesomeStyle).render shouldBe "<i class=\"far fa-grin\"></i>"
```